src/CampaignsAdmin/DropPhotoZone.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { NativeTypes } from 'react-dnd-html5-backend';
+import { useDrop, DropTargetMonitor } from 'react-dnd';
+import { InboxOutlined, DownloadOutlined } from '@ant-design/icons';
+import './dropPhotoZone.css';
+
+interface TargetBoxProps {
+  onDrop: (item: { files: File[] }) => void;
+}
+export const DropPhotoZone: FC<TargetBoxProps> = (props:TargetBoxProps ) => {
```

Comment:
This is more of a preference, but I've seen this instead on multiple teams: 

```tsx
export const DropPhotoZone = (props: TargetBoxProps) => {
```

And reads easier

Reply:
yeah, totally.
I will change it